com.github.diamondburned.gtkcord4.yml

```diff
@@ -0,0 +1,40 @@
+id: com.github.diamondburned.gtkcord4
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command: gtkcord4
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --filesystem=xdg-run/keyring
+  - --device=dri
+
+modules:
+  - name: gtkcord4
+    sources:
```

Comment:
where is the source code?

Reply:
fixed
https://github.com/flathub/flathub/pull/3857/commits/0623091a91d691b5d1330386a5e68c9040ad0b0b#diff-1bd4e3fadd5b9d77e9614182a52a0539614efe5395ebd85821ba7b870a39db0bR35-R37